api/mm/make.build

```diff
@@ -0,0 +1,59 @@
+#------------------------------------------------------------------------------#
+#  DFTB+: general package for performing fast atomistic simulations            #
+#  Copyright (C) 2017  DFTB+ developers group                                  #
+#                                                                              #
+#  See the LICENSE file for terms of usage and distribution.                   #
+#------------------------------------------------------------------------------#
+
+TARGETS = libdftb+.a
```

Comment:
Is the library always intended to be static?

Reply:
No, but with the current build system it's easier this way (shared library would need additional compiler options.)

Reply:
OK, deffer to cmake